dcos_launch/config.py

```diff
@@ -205,6 +205,21 @@ def get_validated_config(user_config: dict, config_dir: str) -> dict:
         'required': True}}
 
 
+def _validate_fault_domain_helper(field, value, error):
+    for fd in value:
+        pass
+
+    if not value.startswith('http'):
+        error(field, 'Not a valid HTTP URL')
+
+
+def _get_num_agents(doc: dict, field_name: str):
+    total = 0
+    for fd in doc.get('fault_domain_helper', list()):
+        total += fd[field_name]
+    return total
```

Comment:
`return sum([fd[field_name] for fd in doc.get('fault_domain_helper', list())] )`?

Reply:
Also, this method doesn't seem to be called anywhere within the new changes 🤔

Reply:
heh, thanks for noticing!